assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSelectionPanel/AudienceCreationErrorNotice.js

```diff
@@ -150,36 +149,27 @@ export default function AudienceCreationErrorNotice( {
 	}
 
 	return (
-		<div className="googlesitekit-audience-creation-error-notice">
-			<WarningSVG width={ 24 } height={ 24 } />
-			<div className="googlesitekit-audience-creation-error-notice__content">
-				{ errorTitle && (
-					<p className="googlesitekit-audience-creation-error-notice__title">
-						{ errorTitle }
-					</p>
-				) }
-				<p className="googlesitekit-audience-creation-error-notice__description">
-					{ errorDescription }
-				</p>
-			</div>
-			{ hasInsufficientPermissionsError && (
-				<div className="googlesitekit-audience-creation-error-notice__actions">
-					<Button
-						href={ requestAccessURL }
-						target="_blank"
-						danger
-						onClick={ () => {
-							trackEvent(
-								`${ viewContext }_audiences-sidebar-create-audiences`,
-								'insufficient_permissions_error_request_access'
-							);
-						} }
-					>
-						{ __( 'Request access', 'google-site-kit' ) }
-					</Button>
-				</div>
-			) }
-		</div>
+		<Notice
+			className="googlesitekit-audience-creation-error-notice"
+			type={ Notice.TYPES.ERROR }
+			title={ errorTitle }
+			description={ errorDescription }
+			ctaButton={
+				hasInsufficientPermissionsError
+					? {
+							label: __( 'Request access', 'google-site-kit' ),
+							href: requestAccessURL,
+							onClick: () => {
+								trackEvent(
+									`${ viewContext }_audiences-sidebar-create-audiences`,
+									'insufficient_permissions_error_request_access'
+								);
+							},
+					  }
+					: null
+			}
```

Comment:
Same here: can't we do `ctaButton={ hasInsufficientPermissionsError ? $OBJECT : undefined }` instead of the object spread?

Reply:
@tofumatt this is already as you describe, there is no object spread on this one.

Reply:
Oh, snap, you're right.

We shouldn't be passing `null` though, it should be `undefined` (the prop types don't specify `null`, and it's probably better for an eventual TypeScript migration not to use both `null` and `undefined` for the same thing here).